src/metabase/util.clj

```diff
@@ -983,3 +983,14 @@
   "Generates a random NanoID string. Usually these are used for the entity_id field of various models."
   []
   (nano-id))
+
+(defn pick-first
```

Comment:
Is this the right place for this function? Are the trade-offs OK?

Reply:
looks fine to me.